core/lib/libmbedtls/include/mbedtls.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
It seems the sole purpose of this file is to be included by `tee_lmd_provider.c` later in this PR.
Any particular reason for that instead of just including the files needed from `tee_lmd_provider.c`?

Reply:
In the beginning of this work, I want to add a middle layer for every algorithm. Because there maybe some structures or macros in those headers files. But now, it seams those header files are not useful anymore. I will remove those files and include them in `tee_lmd_provider.c` directly.